### PR TITLE
Translate Calendar Frontend

### PR DESCRIPTION
**Pull request by stevietv**
#### Database Migration
NO

#### Description
Add translations to Calendar frontend pages

#### Issues Fixed or Closed by this PR

*

### CODE REVIEW

File: frontend/src/Calendar/Agenda/AgendaEvent.js [hunk 1]
```diff
@@ -129,7 +130,7 @@ class AgendaEvent extends Component {
                 <Icon
                   className={styles.statusIcon}
                   name={icons.WARNING}
-                  title="Episode does not have an absolute episode number"
+                  title={translate('MissingAbsoluteNumber')}
```
[the remark this answers is not in the document]
Comment (stevietv) [reply frontend/src/Calendar/Agenda/AgendaEvent.js]: Changed :)

File: frontend/src/Calendar/iCal/CalendarLinkModalContent.js [hunk 2]
```diff
@@ -115,70 +116,70 @@ class CalendarLinkModalContent extends Component {
...
                 value={premieresOnly}
-                helpText="Only the first episode in a season will be in the feed"
+                helpText={translate('ICalSeasonPremieresOnlyHelpText')}
                 onChange={this.onInputChange}
               />
             </FormGroup>
 
             <FormGroup>
-              <FormLabel>Show as All-Day Events</FormLabel>
+              <FormLabel>{translate('ShowAsAllDayEvents')}</FormLabel>
```
Comment (markus101) [inline frontend/src/Calendar/iCal/CalendarLinkModalContent.js:149]: I think we should prefix this with `ICal` same as the help text is.

**Review by markus101 — CHANGES_REQUESTED** [review]
Just the one change then should be good.

File: frontend/src/Calendar/iCal/CalendarLinkModalContent.js (hunk 2 above)
Comment (stevietv) [reply frontend/src/Calendar/iCal/CalendarLinkModalContent.js]: done

**Pull request merged**